source/push-to-deploy.md

```diff
@@ -0,0 +1,86 @@
+- *Memory amount on deploy*: This defines how much memory you want to use to deploy your app.
+- **Environment Variables**: Most common when deploying a Meteor app is `DEPLOY_HOSTNAME`, that you don't need to provide anymore.
+
+<img src="images/push-to-deploy-third-step-advanced-options.png" style="width: 780px;">
+
+Now with all configured you just need to hit **DEPLOY APP** and the first version of your app will be deployed!
+
+<h2 id="setup-up-existing-app">Setting up Push To Deploy for existing app</h2>
+
+On the app page details page you can click on the button `SET UP PUSH TO DEPLOY`.
+
+<img src="images/push-to-deploy-add-config-to-existing-app.png" style="width: 780px;">
+
+The first two steps on this case are the same. You will just have to connect your Git Provider, and then choose the repository. For the last step you will have fewer options change. Basically all you need is to make sure everything is correct on the **Deploy Options** section. As the app is already deployed, all the other fields are already defined.
+
+<img src="images/push-to-deploy-third-step-existing-app.png" style="width: 780px;">
+
+After all it's set up, you can edit it again buy clicking in button `CONFIGURE PUSH TO DEPLOY` in the app details page.
+
+<img src="images/push-to-deploy-add-config-to-existing-app.png" style="width: 780px;">
+
+<h2 id="editing-deploy-settings">Editing next deploy settings</h2>
+
+On Galaxy, in the app version tab, after configuring Push to Deploy to an app, you will find the field `Next Auto Deploy Settings`. Use this field to add [app settings](https://cloud-guide.meteor.com/deploy-guide.html#settings-create) for the next deploy. So when you do the commit to the branch that you configured, these are the settings that will be used. Once you use this field for the first time, the settings will be used every time for the next deploys. If you decide not to use it anymore you can just leave the field empty and save it. 
+
+<img src="images/push-to-deploy-next-app-settings.png" style="width: 780px;">
+
+<h2 id="after-set-up">FAQ</h2>
+
+1 - **How to trigger new deploys?** 
+
+After you finish setting up everything, every commit you do to the chosen branch will trigger a new deploy.
+
+2 - **Can I use the repository and branch to different apps?**
+
+Yes, you can. So be careful when setting up your apps, so you don't deploy the wrong code to an app.
+
+3 - **What happens if a new commit is pushed before the previous deploy is completed?**
+
+The previous deploy will be canceled and the new one will start.
+
+4 - **What docker error 137 means?**
+
+This error means there was an Out Of Memory error. To solve this you should increase the amount of memory you us to deploy your app. You can do this on the third step of the Push to Deploy screen, on the section *Deploy options* > *Advanced options* > *Memory amount on deploy*.
```

Comment:
where is the question explaining that is not possible to cancel?

Reply:
added